roles/configure_ssh_keys/defaults/main.yml

```diff
@@ -1,11 +1,14 @@
-
+    public_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCnmDL26k8/mlYalOw0XZ3PkOLM9EtTQREHhZFVaRjKZPiRi4CnVLrkeEqQo+ojPdxOQSBi/1gSHJttUhvsV1hlrHPK2dwMj/rDrZbbY0/v6UaE8WxQh9JFEunI9b48Q2Y9Rf/7oZ7qPoaMBQ8+Wh1Uhb06HFgiTEW7zkF4GbKH2pB1jTVqVivF+ZpS61zQAxKUGHnvuheq23ALfw66Kp4XDqzwplX+x03BH7SaNhSdI9A8cIe0M/WoLDIgaUF+Gmyb0O2ktinnYnvXnvcV94/AOiuSSyXSwnUhOs7fNL9iwKAJ7JWcjF/QzaYMJ2jEz7+CetKOQ3AE2nbOtkzMzv+7OLo22EcrcnErMaDY6Owt4nM8oc5YvxhdqNQjmUyimrG/f/Yv1tXjs0zz7EvvRe0h8Ziz7iOhFCmCaTIrAtj9wmqG+S6fR5XmQmObZuVvLmGoS4Lm01RcqLS1q3PKrg48e+P0PB/BbQkSLSXq9yL/b6Vi3D0tTCW49jhM7nB60HnVXOV5dAcTm03oAZRAd0s1Ffd3wMuP0mHPF8dKzdn1pl+0it3sxqhkKnOw6tScG9i2B2wV6tBj9VRFKgDyH5t8EVqJzUMzmcJG1ah2kLiFtbaUJz2zwlLDotNLZvT3wmUM+1yhHQidTqSGAHtVi+k8zsYte3OO3V9xuyjXbbwE3Q== john.askew@devinit.org
+  - name: alex
+    public_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD5ZWhy9Mxjf8pIKMM0ZthPNa+ZDD9oWXAEvbRajCFAC5+gCOEx58N/C28S6kekGeUGe1TeL4Chq2PTqWiB5RTapqQghPohsSt+PwvJ2hNq7sjpH5ZaqCwxFtQrFJLnGVZ+FqMSv8d+RH8WTXXY5d6M2xglJ7swfOMb9Rk9QzFXCv0ySFQ06oIdbtOiCxYXFq1nvN5JwqG3f6tvD3GHpu70elGJfRYRoG2YIxdj07i9ULti4s0lhzwUnncWGyJ5lqmTGYiXr7L1UsFKX9vT/NPao3AMRCm0Q1kwtFCAMF8A+mZxm8QCb+S3mbGom8A4323F4HmbLeHnwx9TukiaYzOd alex@mini-beast
+  - name: sam
+    public_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC2EQxjucgMhVlISTBSNJkw/4pcuJN0TjusiMNTdGIlb6cu+T/TddIaSBRiIeVG18udmLL2kvha65h27U+niBBXaSez27pVUQFEE2BHbGpihTDepC1CHU5MhgE/3xQPrTMbhcgc3oIYyRvEaEz+rLIo+uQSF8iXAAMuf9Yj7BBU2MqQ3hZLz6LKOeXssIA4oInIQaaqqn1dBwauvjadaXx7Ysx3fn3bmXbyi7jXJvoBp03IVExjT3FvQTgoYPGaYf1oIpIfiDzn60vrYCHQv8hdibTS7oRNAnze/kYL7sPVokwExqLtHkVv0/1YNgM2TABRe+7Ng0jTYPAlspaeX6Qp6v4alRSYaLLk/wHYpXNHwtPzdt2i3GV86v1bkOsXRQq+oH4NvJkOmfEM6iglvIp7i5QdgfdB9EPxIM0PAbpkt6zFW4+BTSt5keHan+rcnVyBRA0G3Dg6ymE8oLSoJnvKg2h9dboaSgbPv80SSB+AnigA/KvmNYlhfMYyEEe2k2QlCfLLu9Fwc0jNRg8QfzOaL6gWsOcEqPwIFm2RKBbONM2hK5zZ79CmukclBEJVNPt+/ryF6qVu++ZB1kPz5m6O7M1tuOkq4LGADvfufYNA2CdK/2scpp1v/k6e7l3jKm5mk7q7EJCihP5AZXlwitLTWzAZGtw51NWXn6S0H8oY6w== samuele.mattiuzzo@devinit.org
 authorized_keys_external:
   - name: shi
     public_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCiD5YKdszmWOFDHco51vmpDOy+VjXaMQKIwSi7TCCCTFvy5dwqM0DwbHZZEgFHohT1jDqlm2adHeiHn27yv+VKfP9sGFWM7/tPGKpCTt01S/IDwmsFS/ZhgK6KI9W/YntS3QbuHkCso2rNOLDq2fxvlpOXXOMw/qDULGIKTOf+SMxmvjX4KgCQ271dE7L+OGR2wXB8l4G1bzaSFLTAXnxc0ulJATXYYOAAsY2qWhTax5tNG0K2zItiOZWffwmVKw4QwLutTCKW2qqJb8cKgawRWL48FR8cV04A8eShXF/7k9pJfKNMCMRtA1zeRGGC/zTXgyelRrSpzduy/y4ZpBEl notshi@gmail.com
   - name: kris
     public_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDdx+ZGNaTFfqwUzphFSP7m03bu0kLYDuvVg/BXqtimIEPfYTv6K4mZPz7gpsV9OChOX70Y0KHbWVegr6pNUN0h6YxCcOlCRq5/ylifDM08K3SCd/R13MJVvbDuXiBzGflEW2WBPy1zyMEI2SR9lv5PfsRquZnjfDlejn/sLzaZ/PUVHpuweOfGdpKseCM3kh7XhwDAPwGxBx0ckafeiPZjtK0T+yiKR3SXysa2FB5UzW1WBKBqupZXrjgeQ8RC1MKK3MQAPyfiYWT57wsQqyOyPA0oVRKsoez+I3uRWp4KZm9DRya5v231Hw8YuCPWMdPrPUjkafP+RImvBHmEdC1T kriss@xixs.com
 
-unauthorised_keys: []
+unauthorised_keys: []
```

Comment:
You should set your text editor up to handle stuff like this automatically. There should be a new line at the end of this file.